lib/html-min.cls.php

```diff
@@ -166,11 +170,28 @@ public function process()
 		return $this->_html;
 	}
 
+	/**
+	 * From LSCWP 6.2: Changed the function to test for special comments that will be skipped. See: https://github.com/litespeedtech/lscache_wp/pull/622
+	 */
 	protected function _commentCB($m)
 	{
-		return (0 === strpos($m[1], '[') || false !== strpos($m[1], '<!['))
-			? $m[0]
-			: '';
+		// Check if is IE conditional comment
+		$is_ie_comment = (0 === strpos($m[1], '[') || false !== strpos($m[1], '<!['));
+		if($is_ie_comment) return $m[0];
```

Comment:
I think the var $is_ie_comment is redundant.

Reply:
Removed